.github/workflows/build.yml

```diff
@@ -55,6 +55,11 @@ jobs:
         with:
           cli: ${{ env.CLOJURE_VERSION }}
 
+      - uses: actions/checkout@v2
+      - uses: DeLaGuardo/setup-clj-kondo@master
```

Comment:
It seems that yarn lint is already available. Is there any difference?

Reply:
For some reason yarn lint is timing out with the arguments I'm giving clj-kondo. Perhaps this won't be an issue once the card is done e.g. `clj-kondo --lint src`

Reply:
I looked into this a little more and it seems the latest version of the clojure CLI, 1.10.3.1058, hangs with the latest version of clj-kondo. We need the latest version of clj-kondo in order for all lints to be caught correctly. With the last linting PR coming up, I can try to find a workaround and if not, at least log this issue upstream to clj-kondo and put a TODO comment